andrada12/assignment3/WordGridTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+import java.util.ArrayList;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author andrada
+ */
+public class WordGridTest {
+    
+    public WordGridTest() {
```

Comment:
You can leave out the constructor, before and after, they aren't required if you don't use them